File: 1-Iniciante/ex1079.c
#include <stdio.h>

int main(){
    /*variaveis*/
    int i, n; 
    float n1, n2, n3, result;

    /*entrada & processamento & saida*/
    scanf("%d", &n);

    for(i = 0; i < n; i++){
        scanf("%f %f %f", &n1, &n2, &n3);
        result = ((n1 * 2)+(n2 * 3)+(n3 * 5))/(2 + 3 + 5);
        printf("%.1f\n", result);
    }

    return 0;
}

File: 1-Iniciante/ex1079.js
// entrada
let input = require('fs').readFileSync('/dev/stdin', 'utf-8').split('\n'); //adicionar uma barra antes do 'dev' para funcionar no beecrowd
let n = parseInt(input[0]) + 1;

// processamento & saida
for(let i = 1; i < n; i++){
    let [v1, v2, v3] = input[i].split(' ').map(v => parseFloat(v));
    let media_pond = (v1 * 2 + v2 * 3 + v3 * 5) / 10;
    console.log(media_pond.toFixed(1));
}